### PR TITLE
Add Travis CI support

### LINKED ISSUE #4: Add travis_ci tests
this recipe may help:

https://github.com/korfuri/bazel-travis

### PR DESCRIPTION
**Pull request by johnynek**
closes #4

### CODE REVIEW

**Comment by johnynek** [comment]
@dinowernli @damienmg ?

Any concerns?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      - zlib1g-dev
+jdk:
+  - oraclejdk8
+
+before_install:
+  - wget 'https://github.com/bazelbuild/bazel/releases/download/0.2.0/bazel-0.2.0-installer-linux-x86_64.sh'
+  - sha256sum -c .bazel-installer-linux-x86_64.sh.sha256
+  - chmod +x bazel-0.2.0-installer-linux-x86_64.sh
+  - ./bazel-0.2.0-installer-linux-x86_64.sh --user
+  - mv .bazelrc.travis .bazelrc
```
Comment (damienmg) [inline .travis.yml]: why not simply use --bazelrc in the bazel command instead?

**Comment by damienmg** [comment]
LGTM the build passes :+1:

File: .travis.yml (hunk 1 above)
Comment (johnynek) [inline .travis.yml]: we still do the `>>` below. I was just cargo culting this though:

https://github.com/korfuri/bazel-travis/blob/c993a7b5324580619da716f2c2058f7cf65f6ee2/.travis.yml#L24

**Comment by dinowernli** [comment]
LGTM :) Apologies for the delay.

**Pull request merged**